lib/transactional.pm

```diff
@@ -102,6 +106,9 @@ sub process_reboot {
         select_console 'root-console';
         assert_script_run 'clear';
     }
+
+    # Switch to the previous console
+    ($prev_console) ? select_console($prev_console) : die('The $prev_console is not defined.');
```

Comment:
Is this necessary? Is there any situation that `current_console` returns a falsy value?

Reply:
It's not. I'll get rid of it.